templates.dist/fields.php

```diff
@@ -248,96 +248,55 @@ function renderRegionalAddition(string $inputName, array $selectedValue, bool $m
 	function renderParkingLot(array $parkingArray, string $language, string $locale, string $codeCurrency, string $currency): array
 	{
 		$messages = [];
-		foreach ($parkingArray as $key => $parking) {
-		$MarketingType = '';
-		$pluralPaking = '';
-		$preposition = "at";
-			if (!$parking['Count']) {
+		
+
+		foreach ( $parkingArray as $key => $parking ) {
+			$pluralPaking  = '';
+			$preposition   = "at";
+			$MarketingType = $parking['MarketingType'];
+			$parkingName='';
+			if ( ! $parking['Count'] ) {
 				continue;
 			}
 			if ( $parking['Count'] != 1 ) {
 				$pluralPaking = " each";
 			}
-			if ($codeCurrency == "EUR" && $locale == "de_DE" ) {
-				$preposition = "à";
-				$pluralPaking = "";
+
+			$parkingName = getParkingName( $key, $parking['Count'] );
+
+			if ( $codeCurrency == "EUR" && $locale == "de_DE" ) {
```

Comment:
We should translate using the gettext functions (e.g. `__()`). This condition also has the problem that it would not display parking lots in German, just because the estate has a non-euro currency. But the website is in German so it should still be output in German!